### PR TITLE
Cu 2ftp8t9 add method for standard payment link creation on node sdk njoku emmanuel

### PR DESCRIPTION
**Pull request by njokuScript**

### CODE REVIEW

**Comment by njokuScript** [comment]
Task linked: [CU-2ftp8t9 Add method for standard payment link creation on Node SDK](https://app.clickup.com/t/2ftp8t9)

**Review by MeisterJustice — APPROVED** [review]
LG!

File: README.md [hunk 1]
```diff
@@ -8,40 +8,57 @@
...
 ```javascript
-const LazerPay = require('lazerpay-node-sdk');
+const Lazerpay = require('lazerpay-node-sdk');
 
-const lazerpay = new LazerPay(LAZER_PUBLIC_KEY);
+const lazerpay = new Lazerpay(LAZER_PUBLIC_KEY);
 ```
 
 For staging, Use TEST API Keys and for production, use LIVE API KEYS.
 You can get your LAZER_PUBLIC_KEYS from the Lazerpay dashboard.
```
Comment (MeisterJustice) [inline README.md:17]: Change to `Use TEST API keys for testing, and LIVE API keys for production`. 

Reason: Better English, and there's no guarantee that the user has "staging" as environment

File: README.md [hunk 2]
```diff
@@ -8,40 +8,57 @@
...
 
-**1**. **PAYMENT**
+**1**. **Payment**
 
-    * Initialize Payment
-    * Confirm Payments
+- Initialize Payment
+- Confirm Payment
+
+**2**. **Transer**
```
Comment (MeisterJustice) [inline README.md:26]: `Transfer` not Transer

File: README.md (hunk 1 above)
Comment (njokuScript) [reply README.md:17]: cool

File: README.md (hunk 2 above)
Comment (njokuScript) [reply README.md:26]: cool

**Pull request merged**